lib/MusicBrainz/Server/Edit/Alias/Add.pm

```diff
@@ -3,6 +3,7 @@ use MooseX::Role::Parameterized;
 use MooseX::Types::Moose qw( Bool Int Str );
 use MooseX::Types::Structured qw( Dict Optional );
 use MusicBrainz::Server::Data::Utils qw( model_to_type non_empty );
+use MusicBrainz::Server::Data::Utils qw( boolean_to_json );
```

Comment:
`Data::Utils` is already imported right above this line, so just add `boolean_to_json` inside the `qw()` above. (Perl tip: `qw` stands for "quote word" and turns words separated by whitespace into a list of words. so `qw( a b )` is the same as `('a', 'b',)`.)